deploy/templates/notificationtemplates/idlertriggered/notification.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>
+        Notice: Idler is Running on your Developer Sandbox.
+    </title>
+    <style>
+        a:hover {
+            text-decoration: underline !important;
+        }
+        p {
+            text-align: left;
+            margin: 30px 0;
+        }
+    </style>
+</head>
+
+<body
+        style="
+       padding: 10px;
+       padding: 0;
+       background-color: #f9f9f9;
+       font-family: 'Open Sans', sans-serif;
+       font-size: 15px;
+       font-weight: lighter;
+       line-height: 1.2;"
+>
+<div
+        style="
+       min-height: 300px;
+       max-width: 750px;
+       margin: 0 auto;
+       padding: 20px;
+       border: 1px solid #d7d7d7;
+       border-radius: 4px;
+       background-color: #fff;
+       box-shadow: 0 2px 4px #d7d7d7;"
+>
+
+    <p>
+        You are receiving this email because one or more of your applications in the Developer Sandbox for
+        Red Hat OpenShift has been running for 12 hours.
+    </p>
+
+    <p>
+        In accordance with the usage terms of Developer Sandbox, we have reduced the number of instances of your
+        application to zero (0). You can restart your application(s) by increasing the number of instances from the
+        Developer Sandbox User Interface.
```

Comment:
If the user just deployed a pod then I think the idler just deletes the pod so I guess the UI wouldn't have a way to bring it back. The user would have to recreate it so maybe we need to make this message more generic?